x/wasm-storage/keeper/keeper.go

```diff
@@ -35,139 +78,164 @@ func (k Keeper) GetAuthority() string {
 }
 
 // SetDataRequestWasm stores Data Request Wasm using its hash as the key.
-func (k Keeper) SetDataRequestWasm(ctx sdk.Context, wasm *types.Wasm) {
-	store := ctx.KVStore(k.storeKey)
-	bz := k.cdc.MustMarshal(wasm)
-	store.Set(types.GetDataRequestWasmKey(wasm.Hash), bz)
+func (k Keeper) SetDataRequestWasm(ctx sdk.Context, wasm *types.Wasm) error {
+	dataRequestWasmKeyPrefixFull := GetDataRequestWasmKeyPrefixFull(wasm.Hash)
+
+	return k.DataRequestWasm.Set(ctx, dataRequestWasmKeyPrefixFull, *wasm)
```

Comment:
can be merged into single line without declaring a variable

Reply:
[62cfbab](https://github.com/sedaprotocol/seda-chain/pull/192/commits/62cfbabec5a4ad9788cf1ae06c725cf9b2b39a4b)